hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ReopenTableRegionsProcedure.java

```diff
@@ -53,6 +54,16 @@ public class ReopenTableRegionsProcedure
 
   private static final Logger LOG = LoggerFactory.getLogger(ReopenTableRegionsProcedure.class);
 
+  public static final String REOPEN_BATCH_BACKOFF_MILLIS_KEY =
+    "hbase.table.regions.reopen.batch.backoff.ms";
+  public static final long REOPEN_BATCH_BACKOFF_MILLIS_DEFAULT = 0L;
+  public static final String REOPEN_BATCH_SIZE_MAX_KEY =
+    "hbase.table.regions.reopen.batch.size.max";
+  public static final int REOPEN_BATCH_SIZE_MAX_DEFAULT = Integer.MAX_VALUE;
```

Comment:
nitpick: we might want to have these names be `hbase.reopen.table.regions.progressive.batch.size.max` and `backoff.ms`.  It's more of a progressive reopen system now, and i reordered the table.regions.reopen to align more with the procedure name.